site/content/docs/5.0/guidelines/navigation.md

```diff
@@ -270,7 +270,31 @@ toc: true
 </div>
 
 ## Stepped process
+[Documentation]({{< docsref "/components/stepped-process" >}})&nbsp;—&nbsp;
+<span class="d-none d-sm-block d-lg-block">{{< anchor web-nav-stp-001 >}}</span>
+<span class="d-block d-sm-none d-lg-none">{{< anchor web-nav-stp-002 >}}</span>
```

Comment:
Shoudln't we have three IDs here?

Reply:
In Zeplin, "Normal" and "mobile" have the same ID

Reply:
Well, good game GDL :rocket:

Reply:
Only "small" has another ID